src/sidebar/components/sidebar-content.js

```diff
@@ -44,7 +55,7 @@ function SidebarContent({
   });
   const directLinkedTab = linkedAnnotation
     ? tabForAnnotation(linkedAnnotation)
-    : null;
+    : 'annotation'; // default to 'annotation' tab otherwise
```

Comment:
```suggestion
    : 'annotation';
```

The comment seems redundant.